core/src/test/java/org/elasticsearch/test/ESTestCase.java

```diff
@@ -383,6 +383,11 @@ public static String randomRealisticUnicodeOfCodepointLength(int codePoints) {
         }
         return array;
     }
+    
+    public static String randomTimeValue() {
+        final String[] values = new String[]{"d", "H", "ms", "s", "S", "w"};
+        return randomIntBetween(0, 3) + randomFrom(values);
```

Comment:
Lemme clarify I mean that if we add this method to ESTestCase it should be generic enough. I guess there are much more valid values possible for a time value?